clock/app/views/stores/_form.html.erb

```diff
@@ -29,7 +26,13 @@
     </div>
     <div class="col-sm-10">
     <%= form_with(model: store, local: true, class: "form-horizontal") do |form| %>
-            
+    <div class="form-group row">
+      <%= form.label :id, :class => "col-sm-2 col-form-label" %>
```

Comment:
I'm glad to see helper tags from `ActionView`, but I recommend you to use the new syntax
```suggestion
      <%= form.label :id, class: "col-sm-2 col-form-label" %>
```